src/test/java/com/github/dockerjava/core/command/ListContainersCmdImplTest.java

```diff
@@ -104,7 +104,7 @@ public void testListContainersWithLabelsFilter() throws Exception {
         String testImage = "busybox";
 
         // need to block until image is pulled completely
-        dockerClient.pullImageCmd(testImage).exec(new PullImageResultCallback()).awaitCompletion();
+        dockerClient.pullImageCmd(testImage).withTag("latest").exec(new PullImageResultCallback()).awaitCompletion();
```

Comment:
yes, more obvious on a slow network, and it is documented https://docs.docker.com/engine/api/v1.24/
```
tag – Tag or digest. If empty when pulling an image, this causes all tags for the given image to be pulled.```